.github/workflows/build.yaml

```
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.9]
+        python-version: [3.6, 3.8, 3.9]
```

Might as well chuck in 3.7 😁 
```suggestion
        python-version: [3.6, 3.7, 3.8, 3.9]
```